Eventy/src/main/java/org/example/eventy/events/models/Event.java

```diff
@@ -14,12 +16,13 @@ public class Event {
     private EventType type;
     private Location location;
     private List<Activity> agenda;
+    private EventOrganizer organiser;
```

Comment:
Do we really need EventOrganizer in Event class? I think that EventOrganizer has a list of his featured events? We should probably only have one? Let me know what you think!